apps/metadata-editor-e2e/src/e2e/edit.cy.ts

```diff
@@ -1,12 +1,49 @@
+/* eslint-disable cypress/no-unnecessary-waiting */
+
 describe('editor form', () => {
-  beforeEach(() => {
+  let recordUuid
+  before(() => {
     cy.login('admin', 'admin', false)
 
-    // Alpine convention record
-    cy.visit('/edit/8698bf0b-fceb-4f0f-989b-111e7c4af0a4')
-
     cy.clearRecordDrafts()
+    // Duplicate a record to manipulate it at will
+    cy.visit('/catalog/search')
+    cy.get('.table-header-cell').eq(1).click()
+    cy.get('.table-header-cell').eq(1).click()
```

Comment:
Visiting the URL with the search parameters feels more explicit than this double click, which might not work anymore if the order of columns changes.

Reply:
Tried that in the first place, but I think the routing doesn't work properly for that because the columns were ordered in a random way when I did that.